js/flickity.js

```diff
@@ -129,8 +130,16 @@ Flickity.prototype._create = function() {
   this.accel = 0;
   this.originSide = this.options.rightToLeft ? 'right' : 'left';
   // create viewport & slider
-  this.viewport = document.createElement('div');
-  this.viewport.className = 'flickity-viewport';
+  if (this.options.noDomMod) {
+    this.viewport = document.querySelector('.flickity-viewport');
```

Comment:
@oxpa is this working with multiple flickity sliders too?
`querySelector()` references only to the first element found with that _selection_ but if there're more `'.flickity-viewport'` (eg. with more than 1 slider).

I'm just _mind-debugging_ and i think that could :white_check_mark:  work if you run the `querySelector()` in an inner element instead of looking throw all the `document`.